_pages/404.md

```diff
@@ -3,17 +3,16 @@ permalink: /404.html
 title: Page not found
 sitemap: false
 sidenav: false
+common-alert:
```

Comment:
I like this a lot, my only question is what happens if we want to have multiple alerts on a page? We should confirm that that will never happen.

Reply:
We would still be able to add the alert include in the page's md file. The reason I did it this way for this particular alert is that it needs to appear above the page title so I had to add it in the layout.

Reply:
Oh ok gotcha. So for certain alerts the idea is that you could reference the front matter, but we can still manually add variables to other alert includes files?